webapp/components/Modal.vue

```diff
@@ -30,6 +30,17 @@
       :modalData="data.modalData"
       @close="close"
     />
+    <delete-user-modal
+      v-if="open === 'delete'"
+      :id="data.id"
+      :name="data.name"
+      :slug="data.slug"
+      :avatar="data.avatar"
+      :contributionsCount="data.contributionsCount"
+      :commentedCount="data.commentedCount"
+      :createdAt="data.createdAt"
```

Comment:
May add just one parameter with all the needed additional data?
In the new form with callback all these data is stored in `:modalData="data.modalData“`, see above. ☝🏼